src/utils/fireQuery.ts

```diff
@@ -518,10 +518,12 @@ export const getDatalogQuery = (
   args: ReturnType<typeof getDatalogQueryComponents>
 ) => {
   const find = args.definedSelections.map((p) => p.pull).join("\n  ");
-  const query = `[:find\n  ${find}\n:where\n${args.where
-    .map((c) => compileDatalog(c, 0))
-    .join("\n")}\n]`;
-  return query;
+  const where = args.where.map((c) => compileDatalog(c, 0)).join("\n");
+  return `[:find\n  ${find}\n:where\n${
+    args.where.length === 1 && args.where[0].type === "not-clause"
+      ? `[?node :block/uid _]`
+      : ""
+  }${where}\n]`;
```

Comment:
ah just read your shared page - you want `returnNode` which I think should be a part of `args`

Reply:
It actually didn't fix the use case I thought it would (`Evidence` `Supports` `{some individual claim}`) .  

Upon reflection that we currently default to `node`, maybe it is not actually necessary until that design choice is changed.

Out of curiosity for myself, would this be the way to access `returnNode`?
(via adding it to the return type of `getDatalogQueryComponents`)
https://github.com/mdroidian/roamjs-query-builder/commit/8fa4f8f6a3f6ffb0cdbeb32c897f4cd55ec43bef

Reply:
yea we should just reference `returnNode` wherever possible, especially since the query drawer currently makes that configurable.

yup that looks good!